package.json

```diff
@@ -37,51 +37,54 @@
     "ci": "npm run test:once && npm run e2e && npm run lint && npm run build:lib && npm run build:demo-app"
   },
   "dependencies": {
-    "@angular/animations": ">=6.1.0",
-    "@angular/common": ">=6.1.0",
-    "@angular/compiler": ">=6.1.0",
-    "@angular/core": ">=6.1.0",
-    "@angular/forms": ">=6.1.0",
-    "@angular/http": ">=6.1.0",
-    "@angular/platform-browser": ">=6.1.0",
-    "@angular/platform-browser-dynamic": ">=6.1.0",
-    "@angular/router": ">=6.1.0",
-    "@ng-bootstrap/ng-bootstrap": "^2.0.0",
-    "bootstrap-icons": "^1.1.0",
-    "core-js": "^2.5.4",
+    "@angular/animations": "^17.1.1",
+    "@angular/common": "^17.1.1",
+    "@angular/compiler": "^17.1.1",
+    "@angular/core": "^17.1.1",
+    "@angular/forms": "^17.1.1",
+    "@angular/platform-browser": "^17.1.1",
+    "@angular/platform-browser-dynamic": "^17.1.1",
+    "@angular/router": "^17.1.1",
+    "@ng-bootstrap/ng-bootstrap": "^16.0.0",
+    "bootstrap-icons": "^1.11.3",
     "detect-passive-events": "^2.0.3",
-    "rxjs": "^6.5.2",
-    "zone.js": "^0.8.26"
+    "rxjs": "^7.8.1",
+    "tslib": "^2.6.2",
+    "zone.js": "~0.14.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.8.0",
-    "@angular/cli": "~6.2.9",
-    "@angular/compiler-cli": "^6.1.0",
-    "@angular/language-service": "^6.1.0",
-    "@types/jasmine": "~2.8.8",
-    "@types/jasminewd2": "~2.0.3",
-    "@types/node": "14.6.2",
-    "codelyzer": "^4.3.0",
+    "@angular-devkit/build-angular": "^17.1.1",
+    "@angular-eslint/builder": "17.2.1",
+    "@angular-eslint/eslint-plugin": "17.2.1",
+    "@angular-eslint/eslint-plugin-template": "17.2.1",
+    "@angular-eslint/schematics": "17.2.1",
+    "@angular-eslint/template-parser": "17.2.1",
+    "@angular/cli": "^17.1.1",
+    "@angular/compiler-cli": "^17.1.1",
+    "@types/jasmine": "~5.1.4",
+    "@types/node": "^20.11.13",
+    "@typescript-eslint/eslint-plugin": "6.19.0",
+    "@typescript-eslint/parser": "6.19.0",
+    "codelyzer": "^6.0.2",
```

Comment:
It looks like `codelyzer` still pulls in tslint as a dependency. I don't think `codelyzer` is required in any case, so I think it can be removed.

Reply:
Never mind, I have a solution for it on follow-up branch `ng17`.